axes/middleware.py

```diff
@@ -0,0 +1,30 @@
+from axes.attempts import is_already_locked
```

Comment:
This file is not middleware - it should be `backends.py` so the import for this module would be something like `axes.backends.AxesModelBackend`.